lib/backend/syncersv1/felixsyncer/felixsyncer_e2e_test.go

```diff
@@ -87,8 +88,13 @@ var _ = testutils.E2eDatastoreDescribe("Felix syncer tests", testutils.Datastore
 			syncTester.ExpectStatusUpdate(api.ResyncInProgress)
 			syncTester.ExpectStatusUpdate(api.InSync)
 
-			// Add one for the default-allow profile that is always there.
-			expectedCacheSize += 1
+			// Add 2 for the default-allow profile that is always there.
+			// However, no profile labels are in the list because the
+			// default-allow profile doesn't specify labels.
+			expectedProfile := *resources.DefaultAllowProfile()
```

Comment:
Just a nit, but I think it's more idiomatic to do this:

```
expectedProfile := resources.DefaultAllowProfile()
ExpectData(*expectedProfile)
```

Do the ptr dereference when passing it, rather than on the function call.